### DISCUSSION TITLE
Preformance vs sway

### DISCUSSION THREAD

**Discussion by TheRedCyclops** [WillPower3309/swayfx#337] [General]
In your experience how is the performance of swayfx vs sway?

**Comment by WillPower3309** [comment]
We don't have GPU benchmarks, but for Fullscreen applications like games (where it matters most) there should be no difference thanks to direct scanout. I'd guess we're on par with other compositors with eye candy like hyprland since we mostly all use the same or similar shaders

**Comment by markstos** [comment]
I just two effects: rounded corners and drop shadows. I'm not able to perceive a difference.

Which effects you use and how many can have some effect-- for better performance minimize the effects you are using.

**Comment by WillPower3309** [comment]
> I just two effects: rounded corners and drop shadows. I'm not able to perceive a difference.
> 
> Which effects you use and how many can have some effect-- for better performance minimize the effects you are using.  

Blur has effectively zero impact on tiled windows as well. All of our shaders are very well optimized and should give no noticable perf hit :)
